js/admin/carrier_wizard.js

```diff
@@ -792,3 +779,48 @@ function checkAllZones(elt)
 	}
 
 }
+
+/**
+ * Check the carriers range inputs after each change
+ */
+const checkCarriersRangeInput = function checkCarriersRangeInput() {
```

Comment:
We cannot use `const` in legacy ES5 files, ES6 features can only be used in "babelized" files.

Reply:
The whole block for this function is wrongly intended. You should use two spaces for indents.